gnosis/safe-react
# Bugfix - Warning in console regarding stepIndex variable

## What it solves

This PR closes #2429 

## How this PR fixes it

We added `stepIndex` property used in `FullParagraph` as a [Transient](https://styled-components.com/docs/api#transient-props) prop

## How to test it
1. Go to create a Safe
2. Navigate through the steps and you won't see any error in console related to `stepIndex` prop.
